test/aerosol_activation_emulators.jl

```diff
@@ -280,12 +283,116 @@ function get_2modal_model_FT32(;
     end
 end
 
+function calibrate_ARG(
```

Comment:
Lets add it as a separate file and try running in the normal CI (so add it to `runtests.jl`). We moved the emulator tests to longruns, because alltogether they were taking more than 30minutes

Reply:
Or if we want to keep all ML tests together, then you have to manually trigger a buildkite-longruns job. Lmk if you don't know how to do that

Reply:
I was thinking this should stay living with the emulators, but actually I've changed my mind on that now... thanks